scripts/webui/util/imports.py

```diff
@@ -53,6 +53,9 @@
 warnings.filterwarnings("ignore", category=DeprecationWarning)     
 
 defaults = OmegaConf.load("configs/webui/webui_streamlit.yaml")
+if (os.path.exists("userconfig_streamlit.yaml")):
+	user_defaults = OmegaConf.load("userconfig_streamlit.yaml");
+	defaults = OmegaConf.merge(defaults, user_defaults)
```

Comment:
The user config file should be next to the default config and not on the root folder, this would make things more organized and easier to direct users to it if they are having problems with it.